src/contrib/cluster/Akka.Cluster.Sharding/OldCoordinatorStateMigrationEventAdapter.cs

```diff
@@ -17,7 +19,10 @@ internal sealed class OldCoordinatorStateMigrationEventAdapter : IEventAdapter
     {
         public string Manifest(object evt)
         {
-            return "";
+            if (evt is IPersistentRepresentation p)
+                return p.Manifest;
+            
+            return ClusterShardingMessageSerializer.GetManifest(evt);
```

Comment:
Make a comment here - "this should and will throw if `evt` is not a type supported by the ClusterShardingSerializer"

This actually does need to throw.